frontend/src/index.css

```
@@ -1,68 +1,569 @@
+ .py-4 {
+  padding-bottom: 16px;
+  padding-top: 16px;
+}
+ .font-title {
+  font-family:
+    Lato,
+    ui-sans-serif,
+    system-ui,
+    sans-serif,
+    Apple Color Emoji,
+    Segoe UI Emoji,
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Emoji

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)